package.json

```diff
@@ -34,7 +34,7 @@
     "jsftp": "^2.0.0",
     "lodash": "^4.17.4",
     "morgan": "^1.9.0",
-    "node-postal": "^1.0.0",
+    "node-postal": "https://github.com/imothee/node-postal.git",
```

Comment:
hmm, I needed this for testing `nodejs@v12`, @orangejulius what should we do about this?
it only tangentially related to the PR

Reply:
I think it's fine to merge it. I hadn't merged it yet because I figured we'd be merging #146, but its been a while now and Node.js 12 is still not technically supported.